### PR TITLE
Add Generator Options Use-Cases

### PR DESCRIPTION
**Pull request by devstein**
Closes https://github.com/viaduct-ai/kustomize-sops/issues/35

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,6 +177,40 @@ https://github.com/viaduct-ai/kustomize-sops/issues
...
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argocd-secret
+  annotations:
+      # replace the base secret data/stringData values with these encrypted data/stringData values 
+      kustomize.config.k8s.io/behavior: replace
+type: Opaque
+data:
+  # Encrpyted data here
```
Comment (daurnimator) [inline README.md:196]: Typo: s/Encrpyted/Encrypted/

File: README.md [hunk 2]
```diff
@@ -177,6 +177,40 @@ https://github.com/viaduct-ai/kustomize-sops/issues
...
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: argocd-secret
+  annotations:
+      # merge the base secret data/stringData values with these encrypted data/stringData values 
+      kustomize.config.k8s.io/behavior: merge
+type: Opaque
+data:
```
Comment (daurnimator) [inline README.md:210]: Maybe use `stringData` in an example?

File: README.md [hunk 3]
```diff
@@ -177,6 +177,40 @@ https://github.com/viaduct-ai/kustomize-sops/issues
...
+  # Encrpyted data here
```
Comment (daurnimator) [inline README.md:211]: typo again.

File: README.md (hunk 1 above)
Comment (devstein) [reply README.md:196]: Fixed 👍

**Pull request merged**